grc.fraudDetection_web/src/main/java/com/typedb/examples/fraud/db/AddressDao.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright (C) 2022 Vaticle
+ *
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.typedb.examples.fraud.db;
+
+import com.typedb.examples.fraud.model.Address;
+import java.util.Hashtable;
+
+public class AddressDao {
```

Comment:
If we aren't able to use our `StandardDao` generic interface on all `Dao`s, we should look at this again to see if it really is the right approach.

Reply:
We'll say that this is because implementing them all is not usefull for our specific example. But I agree anyway.

Reply:
@SullivanDaly I didn't bother with the CRUD ops here, but I think it makes sense to add them (even if we're not using them at the moment).